dev/NavigationView/NavigationView.xaml

```diff
@@ -547,7 +581,18 @@
                                     </Grid>
                                 </SplitView.Content>
                             </SplitView>
-                            
+
+                            <Grid
```

Comment:
can we make all the updates in this file only for 21h1 if they are not going to be used downlevel ?

Reply:
Unfortunately, a lot of transition and visual states are inherited from SplitView and isn't an explicit visual state on NavView. So I am unable to add those changes via template and it has to be done through codebehind.